### PR TITLE
Added content to Budget Card

### PR DESCRIPTION
**Pull request by ewelina-slepko**

### CODE REVIEW

File: src/app/home/initial-settings/budgets-initial-settings/budgets-initial-settings.component.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  <div class="add-icon-container">
+    <i class="fas fa-plus-circle add-icon" (click)="addBudget()"></i>
+  </div>
+
+  <budgets-form [budgetsList]="budgetsList"
+                [isNewBudgetCardVisible]="isNewBudgetCardVisible"></budgets-form>
+
+  <div class="btn-container">
+    <button standardBtn type="submit" (click)="goToDashboard()">Next</button>
```
Comment (FilipTalaga) [inline src/app/home/initial-settings/budgets-initial-settings/budgets-initial-settings.component.html:23]: Why is that submit? I don't see a form here.

File: src/app/shared/components/card-slider/card-slider.component.ts [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
-  isSwipingLeft = false;
-  isSwipingRight = true;
+  isSwipingRight = false;
+  isSwipingLeft = true;
 
   switchCardOnRight() {
     this.isSwipingLeft = false;
     this.isSwipingRight = true;
-    this.active = this.active < this.walletList.length - 1 ? this.active + 1 : this.active;
+    this.active = this.active > 0 ? this.active - 1 : this.active;
```
Comment (FilipTalaga) [inline src/app/shared/components/card-slider/card-slider.component.ts:22]: You can just:
```ts
this.active -= this.active > 0
```

File: src/app/shared/components/card-slider/card-slider.component.ts [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
   }
 
   switchCardOnLeft() {
     this.isSwipingRight = false;
     this.isSwipingLeft = true;
-    this.active = this.active > 0 ? this.active - 1 : this.active;
+    this.active = this.active < this.walletList.length - 1 ? this.active + 1 : this.active;
```
Comment (FilipTalaga) [inline src/app/shared/components/card-slider/card-slider.component.ts:28]: You can just:
```ts
this.active += this.active < this.walletList.length - 1
```

**Comment by ewelina-slepko** [comment]
Done!

**Pull request merged**